pom.xml

```diff
@@ -82,75 +82,75 @@
+        <version.commons-cli>1.3.1</version.commons-cli>
         <version.commons-codec>1.10</version.commons-codec>
         <version.commons-collections>3.2.2</version.commons-collections>
         <version.commons-digester>1.8</version.commons-digester>
         <version.commons-pool>1.6</version.commons-pool>
         <version.dom4j>1.6.1</version.dom4j>
         <version.gnu.getopt>1.0.13</version.gnu.getopt>
-        <version.groovy-all>2.4.3</version.groovy-all>
+        <version.groovy-all>2.4.7</version.groovy-all>
         <version.httpunit>1.7</version.httpunit>
         <version.io.netty>4.0.35.Final</version.io.netty>
         <version.io.undertow.jastow>2.0.1.Final</version.io.undertow.jastow>
         <version.io.undertow.js>1.0.2.Final</version.io.undertow.js>
         <version.javax.activation>1.1.1</version.javax.activation>
         <version.javax.enterprise>1.2</version.javax.enterprise>
         <version.javax.jws.jsr181-api>1.0-MR1</version.javax.jws.jsr181-api>
-        <version.javax.mail>1.5.5</version.javax.mail>
+        <version.javax.mail>1.5.6</version.javax.mail>
         <version.javax.validation>1.1.0.Final</version.javax.validation>
         <version.jaxen>1.1.6</version.jaxen>
         <version.jboss.jaxbintros>1.0.2.GA</version.jboss.jaxbintros>
-        <version.joda-time>2.9.5</version.joda-time>
+        <version.joda-time>2.9.7</version.joda-time>
         <version.jsoup>1.8.3</version.jsoup>
         <version.net.jcip>1.0</version.net.jcip>
         <version.net.shibboleth.utilities.java-support>7.1.1</version.net.shibboleth.utilities.java-support>
         <version.org.apache.activemq.artemis>1.5.1</version.org.apache.activemq.artemis>
-        <version.org.apache.avro>1.7.6</version.org.apache.avro>
+        <version.org.apache.avro>1.8.1</version.org.apache.avro>
         <version.org.apache.cxf>3.1.8</version.org.apache.cxf>
         <version.org.apache.cxf.xjcplugins>3.0.5</version.org.apache.cxf.xjcplugins>
         <version.org.apache.ds>2.0.0-M20</version.org.apache.ds>
-        <version.org.apache.httpcomponents.httpasyncclient>4.1</version.org.apache.httpcomponents.httpasyncclient>
+        <version.org.apache.httpcomponents.httpasyncclient>4.1.2</version.org.apache.httpcomponents.httpasyncclient>
         <version.org.apache.jstl>1.2.5</version.org.apache.jstl>
         <version.org.apache.james.apache-mime4j>0.6</version.org.apache.james.apache-mime4j>
         <version.org.apache.lucene>5.3.1</version.org.apache.lucene>
-        <version.org.apache.myfaces.core>2.2.10</version.org.apache.myfaces.core>
+        <version.org.apache.myfaces.core>2.2.11</version.org.apache.myfaces.core>
         <version.org.apache.openjpa>2.4.1</version.org.apache.openjpa>
         <version.org.eclipselink.version>2.6.3</version.org.eclipselink.version>
         <version.org.apache.neethi>3.0.3</version.org.apache.neethi>
         <version.org.apache.qpid.proton>0.8</version.org.apache.qpid.proton>
```

Comment:
@rsvoboda yes, it's safe in this specific case, as you noticed we've already tested the combination with jbossws-cxf master.